app/src/main/java/org/simple/clinic/home/overdue/OverdueEffectHandler.kt

```diff
@@ -25,8 +26,8 @@ class OverdueEffectHandler @AssistedInject constructor(
     return RxMobius
         .subtypeEffectHandler<OverdueEffect, OverdueEvent>()
         .addTransformer(LoadCurrentFacility::class.java, loadCurrentFacility())
-        .addTransformer(LoadOverdueAppointments::class.java, loadOverdueAppointments())
-        .addConsumer(OpenContactPatientScreen::class.java, { uiActions.openPhoneMaskBottomSheet(it.patientUuid)}, schedulers.ui())
+        .addConsumer(LoadOverdueAppointments::class.java, ::loadOverdueAppointments, schedulers.ui())
```

Comment:
Doesn't data source factory change the thread when we attach to paging adapter? Do we have manually change the thread here?

Reply:
Hmm, I am not sure about this. I added the thread switching out of habit.

Reply:
Yeah, we will need to call it on the UI thread because the data source internally subscribes to the screen detaches which needs to be invoked on the UI thread.